File: Solutions/LeeInkyu/Game Shop/04/Item.java
/**
 * Created by Inkyu Lee on 2017-05-28.
 */
public class Item {
    String name, description;
    int weight, value;

    public Item() {}

    public Item(String namePm, String descriptionPm, int weightPm, int valuePm)
    {
        this.name = namePm;
        this.description = descriptionPm;
        this.weight = weightPm;
        this.value = valuePm;
    }

    public void describe()
    {
        System.out.println("Name        = " + this.name);
        System.out.println("Description = " + this.description);
        System.out.println("Weight      = " + this.weight + " lbs");
        System.out.println("Value       = " + this.value + " gold coins");
    }
}

File: Solutions/Stareat0707/Game Shop/Game Shop, Part 1/Item.cpp
#include "Item.h"

Item::Item(std::string name, std::string description, int weight, int value) : name(name), description(description), weight(weight), value(value)
{
}

void Item::Describe()
{
	std::cout << "Name          = " << name << std::endl;
	std::cout << "Description   = " << description << std::endl;
	std::cout << "Weight        = " << weight << " lbs" << std::endl;
	std::cout << "Value         = " << value << " coins" << std::endl;
}